tsdb/head_bench_test.go

```diff
@@ -79,6 +86,71 @@ func BenchmarkHeadStripeSeriesCreate_PreCreationFailure(b *testing.B) {
 	}
 }
 
+func BenchmarkHead_WalCommitWithVaryingSeriesSamplesExemplarsAndHistograms(b *testing.B) {
+	minute := func(m int) int64 { return int64(m) * time.Minute.Milliseconds() }
+	seriesCounts := []int{100, 1000, 10000}
+	series := genSeries(10000, 10, 0, 0)
+	histograms := genHistogramSeries(10000, 10, minute(0), minute(119), minute(1), true)
```

Comment:
```suggestion
	floatHistograms := genHistogramSeries(10000, 10, minute(0), minute(119), minute(1), true)
	histograms := genHistogramSeries(10000, 10, minute(0), minute(119), minute(1), false)
```

What do you think about appending normal histograms as well? Just for completeness sake :P